contrib/runners/inquirer/tests/unit/test_inquirer_runner.py

```diff
@@ -0,0 +1,153 @@
+        runner = inquirer.get_runner()
+        runner.context = {
+            'user': test_user
+        }
+        runner.action = self._get_mock_action_obj()
+        runner.runner_parameters = runner_params
+        runner.container_service = service.RunnerContainerService()
+        runner.pre_run()
+        mock_liveaction_db.context = {
+            "parent": test_parent_id
+        }
+        (status, output, _) = runner.run({})
+        self.assertEqual(status, LIVEACTION_STATUS_PENDING)
+        self.assertEqual(
+            output,
+            {
+                'users': [],
+                'roles': [],
+                'tag': "developers",
+                'schema': {},
+                'ttl': 1440
+            }
+        )
+        mock_trigger_dispatcher.return_value.dispatch.assert_called_once_with(
+            'core.st2.generic.inquiry',
+            {
+                'users': [],
+                'roles': [],
+                'id': None,
+                'tag': "developers",
+                'ttl': 1440,
+                'schema': {}
+            }
+        )
+        mock_action_service.request_pause.assert_called_once_with(
+            test_parent,
+            test_user
+        )
+        mock_trigger_dispatcher.reset_mock()
+        mock_action_utils.reset_mock()
+        mock_action_service.reset_mock()
+
+    @mock.patch('inquirer.TriggerDispatcher', mock_trigger_dispatcher)
+    @mock.patch('inquirer.action_utils', mock_action_utils)
+    @mock.patch('inquirer.action_service', mock_action_service)
+    def test_inquiry_no_parent(self):
+        """Should behave like a regular execution, but without requesting a pause
+        """
+
+        runner = inquirer.get_runner()
+        runner.context = {
+            'user': 'st2admin'
+        }
+        runner.action = self._get_mock_action_obj()
+        runner.runner_parameters = runner_params
+        runner.container_service = service.RunnerContainerService()
+        runner.pre_run()
+        mock_liveaction_db.context = {
+            "parent": None
+        }
+        (status, output, _) = runner.run({})
+        self.assertEqual(status, LIVEACTION_STATUS_PENDING)
+        self.assertEqual(
+            output,
+            {
+                'users': [],
+                'roles': [],
+                'tag': "developers",
+                'schema': {},
+                'ttl': 1440
+            }
+        )
+        mock_trigger_dispatcher.return_value.dispatch.assert_called_once_with(
+            'core.st2.generic.inquiry',
+            {
+                'users': [],
+                'roles': [],
+                'id': None,
+                'tag': "developers",
+                'schema': {},
+                'ttl': 1440
+            }
+        )
+        mock_action_service.request_pause.assert_not_called()
+        mock_trigger_dispatcher.reset_mock()
+        mock_action_utils.reset_mock()
+        mock_action_service.reset_mock()
+
```

Comment:
How about test inquiry w/ a grandparent?

Reply:
Since `action_service` is being mocked out for these tests, adding another test for testing this would seemingly prove nothing - the mock will obviously return whatever I tell it to. As you correctly point out, the `action_service.get_root_liveaction()` could use some dedicated testing, so I'll test for complex inputs like this there.

Reply:
Added test for the underlying function in 5c9be2438